tests/phpunit/Civi/API/V4/ConformanceTest.php

```diff
@@ -46,8 +48,21 @@ public function testConformance() {
       $this->report("Actions: \n".json_encode(
         array_keys((array)$actions),JSON_PRETTY_PRINT));
 
-
       if ($entity != 'Entity') {
+        // FIELDS ////////////////////
+        $fields = $entity_class::getFields()
+          ->setCheckPermissions(FALSE)
+          ->execute()->indexBy('name');
```

Comment:
I think each method should be on a new line. Are you running `civilint` before committing?